herbie/main.rkt

```diff
@@ -13,7 +13,9 @@
 (require "alt-table.rkt")
 (require "matcher.rkt")
 
-(provide improve)
+(provide improve
+	 ;; For the shell
+	 (all-defined-out))
```

Comment:
We'll want to fix this later, but for now this is fine. I'd rather merge in technical debt and then clear it together with other technical debt.